### DISCUSSION TITLE
Different folder colors

### DISCUSSION THREAD

**Discussion by acchs** [SylEleuth/gruvbox-plus-icon-pack#128] [Ideas]
I just wanted to share some thoughts regarding the different colored folders.
I noticed over a year ago in #90, SylEleuth mentioned a script wasn't implemented, since a method isn't known. As far as I know, `ln` can't provide that functionality. Most projects I've looked at only provide the option to change colors at compilation/installation/setup, which simplifies the process, since the folder structure would be known at that time. 
Personally, I think providing a script that can be run in the `Gruvbox-Plus-Dark/places/scalable/` to change folder colors would be helpful, although it wouldn't be much more impactful than the script in #89. Having a simple script linked to in the README, or even adding #89 to README would be very helpful.
Also, only allowing the user to modify the colors at compilation or requires a certain folder structure doesn't sit well with me, since it's less friendly for modding.

**Comment by SylEleuth** [comment]
Today I started uploading new folder icons, it will take some time. They will get slightly new look and structures of files will change. So it will be advice to create a script after I'll finish uploading. I will also add separate directories for each colour which will make a little bit easier to change the whole folder colours. But it won't be a part of icon pack and it will be required to download (clone) separately. Until someone will write a proper script.
Right now You've got a new colour group called `plasma` which is not a colour but is blue as default. It's actually a set of icons created specifically for KDE Plasma users. They can changed folder's colour in system settings.

**Reply by acchs** [reply]
Sorry if I'm replying a little late, but I just thought of an idea that might work for a structure.
1. Make a folder for each icon color (Folder-Purple for example)
2. Make a link to the selected folder color folder; for example:
`ln -sf Folder-Purple Folder-General`
3. Link each icon to an icon in Folder-General (just a placeholder name)
ex: places/scalable/folder.svg -> Folder-General/folder.svg (-> Folder-Purple/folder.svg)

This allows the user to switch the color with just a simple command: ln -sf Folder-COLOR Folder-General

**Reply by SylEleuth** [reply]
It will be much simpler https://github.com/SylEleuth/gruvbox-plus-icon-pack/pull/145
